src/edu/csus/ecs/pc2/clics/API202003/ResourceConfig202003.java

```diff
@@ -0,0 +1,119 @@
+        }
+
+        if (wsPropUtilities.getBooleanProperty(WebServerPropertyUtils.FETCH_RUN_SERVICE_ENABLED_KEY, false)) {
+            resConfig.register(new FetchRunService(getContest(), getController()));
+            showMessage("Starting /fetchRun web service");
+        }
+
+        // CLICS Contest API services are collective -- either all enabled or all disabled (and default to enabled if unspecified)
+        if (wsPropUtilities.getBooleanProperty(WebServerPropertyUtils.CLICS_CONTEST_API_SERVICES_ENABLED_KEY, true)) {
+
+            resConfig.register(new ContestService(getContest(), getController()));
+            showMessage("Starting /contest web service");
+            resConfig.register(new ScoreboardService(getContest(), getController()));
+            showMessage("Starting /contest/scoreboard web service");
+            resConfig.register(new LanguageService(getContest(), getController()));
+            showMessage("Starting /contest/languages web service");
+            resConfig.register(new TeamService(getContest(), getController()));
+            showMessage("Starting /contest/teams web service");
+            resConfig.register(new GroupService(getContest(), getController()));
+            showMessage("Starting /contest/groups web service");
+            resConfig.register(new OrganizationService(getContest(), getController()));
+            showMessage("Starting /contest/organizations web service");
+            resConfig.register(new JudgementTypeService(getContest(), getController()));
+            showMessage("Starting /contest/judgement-types web service");
+            resConfig.register(new ClarificationService(getContest(), getController()));
+            showMessage("Starting /contest/clarifications web service");
+            resConfig.register(new SubmissionService(getContest(), getController()));
+            showMessage("Starting /contest/submissions web service");
+            resConfig.register(new ProblemService(getContest(), getController()));
+            showMessage("Starting /contest/problems web service");
+            resConfig.register(new JudgementService(getContest(), getController()));
+            showMessage("Starting /contest/judgements web service");
+            resConfig.register(new RunService(getContest(), getController()));
+            showMessage("Starting /contest/runs web service");
+            resConfig.register(new EventFeedService(getContest(), getController()));
+            showMessage("Starting /contest/event-feed web service");
+            resConfig.register(new StateService(getContest(), getController()));
+            showMessage("Starting /contest/state web service");
+            resConfig.register(new VersionService(getContest(), getController()));
+            showMessage("Starting / endpoint for version web service");
+
+        }
+
+        return resConfig;
+    }
+
+    // Maybe these should be in a base class that all API's inherit?
```

Comment:
This sounds like a good idea.  The "private" methods would of course have to be changed to "protected"...